riptide-stream/README.md

```diff
@@ -1,7 +1,7 @@
 # Riptide: Streams Extension
```

Comment:
Yes you are right, but I didn't like users to depend on this. It's result of the problem, that I could not find an easy solution. removing the value using the JsonParser itself, but I might find in the future.